doc/guide/cockpit-file.xml

```diff
@@ -8,10 +8,14 @@
   </refnamediv>
 
   <refsynopsisdiv>
+    <para>The <code>cockpit.file</code> API lets you read, write, and watch files in their entirety.  It can not
+    efficiently do random access in a big file, for example, or read streams such as <code>/dev/random</code>.
+    </para>
```

Comment:
Is it worth noting that we only open `regular` files? Entries in `/proc` are also part of that, right?
```
stat /proc/meminfo 
  File: '/proc/meminfo'
  Size: 0         	Blocks: 0          IO Block: 1024   regular empty file
```